_pages/en_US/dumping-games-and-carts.txt

```diff
@@ -0,0 +1,140 @@
+---
+title: "Dumping Titles and Game Cartridges"
+---
+
+{% include toc title="Table of Contents" %}
+
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
+{: .notice--primary}
+
+### Required Reading
+
+Aside from creating and restoring NAND backups, GodMode9 has the functionality to dump installed titles to an installable `.cia` file, to dump game cartridge data to a `.3ds` ROM file, and/or to directly install a game cartridge to the system.
+
+## Updating GodMode9
+
+These instructions are written for use with GodMode9 v2.0.0 or later. If you have an older version of GodMode9, follow these instructions to update it.
+{: .notice--info}
+
+### What You Need
+
+* The latest release of [GodMode9](https://github.com/d0k3/GodMode9/releases/latest)
+
+### Instructions
+
+1. Power off your device
+1. Insert your SD card into your computer
+1. Copy `GodMode9.firm` from the GodMode9 `.zip` to the `/luma/payloads/` folder on your SD card
+1. Copy the `gm9` folder from the GodMode9 `.zip` to the root of your SD card
+1. Reinsert your SD card into your device
+
+## Dumping a Game Cartridge
+
+{% capture notice %}
+Insert the game cartridge you intend to dump into your device
+  + 3DS game cartridges will be dumped to a `.3ds` format
+  + NDS game cartridges will be dumped to a `.nds` format
+{% endcapture %}
+
+<div class="notice--info">{{ notice | markdownify }}</div>
+
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Navigate to `[C:] GAMECART`
+1. Follow the steps applicable to your game cartridge:
+  + **3DS Game Cartridge:** Press (A) on `[TitleID].trim.3ds` to select it
+  + **NDS Game Cartridge:** Press (A) on `[TitleID].trim.nds` to select it
+1. Select "Copy to 0:/gm9/out"
+1. Your non-installable `.3ds` or `.nds` formatted file will be outputted to the `/gm9/out/` folder on your SD card
+
+## Installing a Game Cartridge Directly to the System
+
+This will only work for 3DS games; it is not possible to install an NDS game cartridge to the system as a title.
+{: .notice--info}
+
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Navigate to `[C:] GAMECART`
+1. Press (A) on `[TitleID].trim.3ds` to select it, then select "NCSD image options...", then select "Install game image"
+1. Press (A) to unlock SysNAND (lvl1) writing, then input the key combo given
+1. Your game will now show up in the HOME menu as an installed title
```

Comment:
Maybe use "Once the process is complete, you game will show up in the HOME Menu as an installed title" to minimize any chance of people interrupting this